locations/spiders/ultra_liquors.py

```diff
@@ -0,0 +1,30 @@
+import scrapy
+
+from locations.categories import Extras, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.user_agents import BROWSER_DEFAULT
+
+
+class UltraLiquorsSpider(scrapy.Spider):
+    name = "ultra_liquors"
+    item_attributes = {"brand": "Ultra Liquors", "brand_wikidata": "Q116620602"}
+    allowed_domains = ["greenpoint.ultraliquors.co.za"]
+    user_agent = BROWSER_DEFAULT
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+
+    def start_requests(self):
```

Comment:
Possibly a FormRequest is not needed and Request would suffice? If there is a cut-off of results below 1000 unless `length` is specified, I think this FormRequest may need a `method="POST"` parameter too?

Currently all 23 results are returned with a straight no-parameter Request to the supplied URL, thus `start_requests` may not need to be overridden, and `start_urls` could be set instead?

Reply:
I am seeing that only 10 results being returned when no `length` parameter is sent but the response is misleading as it says there are 23 results.

`"POST"` is the default method set by `FormRequest` if one is not supplied. Should we set it anyway?